devel/nixpacks/Portfile

```diff
@@ -15,9 +15,9 @@ long_description    Nixpacks takes a source directory and produces an OCI compli
 homepage            https://nixpacks.com/
 
 checksums           ${distname}${extract.suffix} \
-                    rmd160  066cf0a3f9d1d3beec4344e3caafa27c144f4500 \
-                    sha256  f0dcda2cc973ab3c621c48e824b1f068e690c67fddfd2cfabd66a27bd930a046 \
-                    size    30567225
+					rmd160  728c994c001c5386a6c5b3c369675a6c67086aeb \
```

Comment:
This may seem like a nit, but the indentation has been messed up in the patch. Fix that and we can commit the update immediately.

Reply:
Done.
Vi messes files sometimes on copy/paste.